internal/build/tar_test.go

```diff
@@ -56,6 +56,24 @@ func TestArchivePathsIfExists(t *testing.T) {
 	f.assertFileNotInTar(actual, "b")
 }
 
+func TestLen(t *testing.T) {
+	ab := newArchiveBuilder()
+	dfText := "FROM alpine"
+	df := Dockerfile(dfText)
+	err := ab.archiveDf(context.Background(), df)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	ab.close()
+	expected := 2048
```

Comment:
this is just sort of a golden master test. I couldn't think of a better way to do this.